src/components/Modal/Down.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { useRef, useState } from 'react';
+import styled from 'styled-components';
+import { TiArrowSortedUp, TiArrowSortedDown } from 'react-icons/ti';
+
+function Down(props) {
+  const [isOpen, setIsOpen] = useState(false);
+  const [change, setChange] = useState(false);
+
+  const parentRef = useRef();
+  // const [clicked, setClicked] = useState(false);
+  // const toggle = onClick => {
+  //   if (clicked === onClick) {
+  //     return setClicked(null);
+  //   }
+  //   setClicked(Category);
+  // };>>>>>
+
+  return (
+    <div className="DropDown">
+      <Button
+        onClick={() =>
+          setIsOpen(!isOpen, setChange(change === false ? true : false))
+        }
+      >
+        <Title>{props.label}</Title>
+        {change ? <TiArrowSortedUp /> : <TiArrowSortedDown />}
+      </Button>
+      <ContentPrent
+        ref={parentRef}
+        style={
+          isOpen
+            ? {
+                height: parentRef.current.scrollHeight + 'px',
+              }
+            : {
+                height: '0px',
+              }
+        }
+      >
```

Comment:
height 내에서 삼항연산자를 사용할 수 있을 것 같네요.
```suggestion
      <ContentPrent
        ref={parentRef}
        style={{ height: isOpen ? "  " : "  " } }
      >
```